.github/workflows/e2e.yml

```diff
@@ -234,17 +234,17 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        node: [16, 18, 20]
+        node: [20, 22]
         include:
           - os: ubuntu-latest
-            container: ubuntu:16.04
-            node: 16
+            container: ubuntu:20.04
+            node: 20
           - os: ubuntu-latest
-            container: fedora:24
-            node: 16
+            container: fedora:29
```

Comment:
```suggestion
            container: fedora:28
```

Reply:
fedora:28 comes with a glibc27, but we're targeting to glibc28 as a minimum

Reply:
It seems you are right. Let's rollback